tests/address_test.go

```diff
@@ -86,6 +86,22 @@ func (c *ClientTests) TestAddressCreateVerify() {
 	assert.Equal("417 MONTGOMERY ST STE 500", address.Street1)
 }
 
+func (c *ClientTests) TestAddressCreateAndVerify() {
+	client := c.TestClient()
+	assert := c.Assert()
+
+	address, _ := client.CreateAndVerifyAddress(
+		c.fixture.IncorrectAddressToVerify(),
+		&easypost.CreateAddressOptions{
```

Comment:
This should probably be on the fixture level as it is in other libs, you may need to remove other tests that have the option in the test itself and just move it to the fixture: https://github.com/EasyPost/easypost-php/blob/b654b3e9b43ef37992727fb48b8799df99dd5a98/test/EasyPost/Fixture.php#L63

Reply:
This would roll back to the previous commit code, is this fine?

`
Verifications: &easypost.AddressVerifications{
			Delivery: &easypost.AddressVerification{
				Success: true,
       }
}
`

Reply:
After talking, I misunderstood. This is correct and because the options is a parameter, we can't include it in the fixture.